arithmetic-operators.py

```diff
@@ -0,0 +1,6 @@
+""" 
+This python program is to demonstrate the use of basic arithmetic operators in python.
+"""
+num1=int(input("Enter first number: "))
+num2=int(input("Enter second number: "))
+print(num1,'+',num2,'=',num1+num2)
```

Comment:
There should be proper print statement. like The sum of number...

Reply:
@mohanpoddar I have made the suggested changes. Please review.